analyzer/src/test/kotlin/MavenSupportTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.analyzer
+
+import com.here.ort.model.VcsInfo
+
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+
+import org.apache.maven.model.Scm
+import org.apache.maven.project.MavenProject
+
+class MavenSupportTest : WordSpec({
+    "parseVcsInfo()" should {
+        "handle GitRepo URLs" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    connection = "scm:git-repo:ssh://host.com/project/foo?path/to/manifest.xml"
```

Comment:
Is this a documented way to write Git-Repo URLs, or is it something we made up?

Reply:
It's something that I made up to work around a concrete issue that we currently have. See http://maven.apache.org/scm/scms-overview.html for the list of officially supported SCMs.

Reply:
Let's discuss this separately when it's more clear if we need to implement case 2), which I'd rather not support, as for me git-repo manifests simply belong into Git repos.

Reply:
Agree, makes sense.